db/migrate/20161109162634_add_origin_to_epic_queue_record.rb

```diff
@@ -0,0 +1,6 @@
+class AddOriginToEpicQueueRecord < ActiveRecord::Migration
+  def change
+    add_column :epic_queue_records, :origin, :string
+    add_column :epic_queue_records, :identity_id, :integer
```

Comment:
Meh, as far as I can tell, the actual end result is identical in this instance, but I'll change it just for consistency.